common/src/chain/block/height_skip.rs

```diff
@@ -0,0 +1,1032 @@
+use crate::primitives::BlockHeight;
+
+fn invert_lowest_one(n: u64) -> u64 {
+    n & n.wrapping_sub(1)
+}
+
+pub fn get_skip_height(h: BlockHeight) -> BlockHeight {
```

Comment:
Would this be more ergonomic/idiomatic as a method of `BlockHeight`?

Reply:
I don't prefer that. It's unrealted to the block height concept.

Reply:
Fair enough